### PR TITLE
add a script to build local repo, instruct user to deploy to cloudfoundry

### PR DESCRIPTION
**Pull request by datianshi**

### CODE REVIEW

File: build_local.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+echo "GOPATH is set as: $GOPATH"
+
+PROJECT_DIR="$GOPATH/src/github.com/pivotalservices/ignition"
```
Comment (zmb3) [inline build_local.sh]: `$GOPATH` is not mandatory anymore.  Can we replace this with `$(go env GOPATH)` instead?

File: build_local.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+go test ./...
+pushd $PROJECT_DIR/cmd/ignition
+  GOOS=linux GOARCH=amd64 go build -o $BUILD_DIR/ignition-linux
+  GOOS=darwin GOARCH=amd64 go build -o $BUILD_DIR/ignition-mac
+  GOOS=windows GOARCH=amd64 go build -o $BUILD_DIR/ignition-win64
+popd
+
+pushd $PROJECT_DIR/web
+  yarn lint && yarn test && yarn build
```
Comment (zmb3) [inline build_local.sh]: We should probably include `yarn install` just to be safe.

Comment (datianshi) [reply build_local.sh]: yarn install already taken care on the setup.sh

File: build_local.sh (hunk 1 above)
Comment (datianshi) [reply build_local.sh]: will do

**Comment by datianshi** [comment]
@zmb3 squashed the commit with $(go env GOPATH)

**Pull request merged**